cmd/cluster/core/create.go

```diff
@@ -329,6 +330,16 @@ func prototypeResources(opts *CreateOptions) (*resources, error) {
 		},
 	}
 
+	if len(opts.DisableClusterCapabilities) > 0 {
+		caps := make([]hyperv1.OptionalCapability, len(opts.DisableClusterCapabilities))
+		for i, c := range opts.DisableClusterCapabilities {
+			caps[i] = hyperv1.OptionalCapability(c)
+		}
+		prototype.Cluster.Spec.Capabilities = &hyperv1.Capabilities{
+			DisabledCapabilities: caps,
```

Comment:
This name is not aligned with the spec in the API.

Reply:
Thanks @jparrill , I updated it in this PR by accident - it was meant to go in https://github.com/openshift/hypershift/pull/5810. I just fixed it. I'll put this PR on hold so the other can merge. I'll make sure to rebase once that happens.